README.md

```diff
@@ -44,6 +44,10 @@ Install
     ```bash
     composer require baopham/dynamodb
     ```
+* For Lumen only - additional steps
+  - enable eloquent support in bootstrap/app.php: uncomment/add `$app->withEloquent()` just after `$app = new Laravel\Lumen\Application ...`
+  - set up a `config_path` helper: https://gist.github.com/mabasic/21d13eab12462e596120
```

Comment:
Looks like you just need to do `$app->configure('dynamodb');`? See #134

Reply:
Yes, but that file need to be copied from the src folder of the package.  Manually or using `vendor-publish`. I prefer the latter, hence more steps.